multi_host_migration/tests/cfg/multi_host.cfg

```diff
@@ -75,6 +75,7 @@
                     paused_after_start_vm = yes
                     preprocess_env = yes
                     enable_random_timeout = yes
+                    start_migration_timeout = random
```

Comment:
just see "start_migration_timeout" used in code, so add in cfg file. I will update them. Many thanks